manual/manual/html_processing/src/process_manual.ml

```diff
@@ -0,0 +1,370 @@
+(* Post-processing the HTML of the OCaml Manual.
+
+   San Vu Ngoc, 2020
+
+   * Processed parts: ["tutorials"; "refman"; "commands"; "library" ]
+
+   * The "API" side is treated by another script.
+
+   requires Lambdasoup
+
+   cd ..; make web
+
+ *)
+
+open Soup
+open Printf
+open Common
+
+(* Alternative formats for the manual: *)
+let archives =
+  ["refman-html.tar.gz"; "refman.txt"; "refman.pdf"; "refman.info.tar.gz"]
+
+(* Remove number: "Chapter 1  The core language" ==> "The core language" *)
+let remove_number s =
+  Str.global_replace (Str.regexp ".+  ") "" s
+
+(* Scan index.html and return the list of chapters (title, file) *)
+let index part =
+  sprintf "Reading part [%s]" part |> pr;
+  let html = read_file (html_file "index.html") in
+  let soup = parse html in
+  (* Foreword. We do nothing. *)
+  (* We select the list of (html files, titles) for Part [part] *)
+  let part =
+    let a = match select_one ("a[id=\"p:" ^ part ^ "\"]") soup with
+      | Some node -> node
+      | None -> R.select_one ("a[name=\"p:" ^ part ^ "\"]") soup in
+    let ul = next a in
+    assert (name ul = "ul");
+    ul $$ "a"
+    |> fold (fun list a ->
+        (R.leaf_text a |> remove_number, R.attribute "href" a) :: list) []
+    |> List.rev in
+  part
+
+
+(* will be automatically updated *)
```

Comment:
how/when?

Reply:
you're right, this was a copy-paste from the `process_api.ml`, but in the current case, it was more complicated to try to update this string, and I finally gave up. Let's remove this comment.

Reply:
It's just `\number\year` in the TeX files - why not just use `Unix.gettimeofday`?